drivers/gpu/drm/v3d/Kconfig

```diff
@@ -1,6 +1,6 @@
 config DRM_V3D
 	tristate "Broadcom V3D 3.x and newer"
-	depends on ARCH_BCM || ARCH_BCMSTB || COMPILE_TEST
+	depends on ARCH_BCM || ARCH_BCMSTB || ARCH_BCM2835 || COMPILE_TEST
```

Comment:
The arm64 bcm2711_defconfig should defiine ARCH_BCM=y, like the 32-bit version.

Reply:
ARCH_BCM wasn't defined at all for arm64, so I added it.

Reply:
Please keep your original change by adding ARCH_BCM2835.

Reply:
Do you still want ARCH_BCM to be defined?

Reply:
I don't think this is necessary.